File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int x = in.nextInt(), y = in.nextInt();
        int res;
        if(x > 0 || y > 0){
            res= x > 0 && y > 0 ? 1 : x > 0 ? 4 : 2;
        }
        else
            res = 3;
        System.out.print(res);
            
    }
}

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var x = sc.nextInt(); var y = sc.nextInt()
    var res = when{
        x > 0 && y > 0 -> 1
        x > 0 -> 4
        y > 0 -> 2
        else -> 3
    }
    print(res)
}